orbitx/flight_gui.py

```diff
@@ -14,9 +14,11 @@
 import math
 
 from . import common
-from . import orbitx_pb2 as protos
+from . import orbitx_pb2 as protos  # physics module
 
-import vpython
+import vpython                      # python 3D graphic library module
+import orbitx.style as style        # HTML5, Javascript and CSS3 code for UI
```

Comment:
is there some way we can have HTML and CSS code be in .html and .css files? I'm not the web programmer, you'll probably know more than me, but if we're rearranging files, it might be even better practice to put HTML and CSS code in their own files instead of in a python file, since it'll be completely decoupled!

Reply:
Ok. In the meantime, if you're comfortable with this we can just merge it as it is, since it's a much better structure right now.